jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererFileContextProvider.kt

```diff
@@ -110,7 +112,11 @@ class DefaultCodeWhispererFileContextProvider(private val project: Project) : Fi
         val isTst = isTestFile(psiFile)
 
         val chunks = if (isTst && targetContext.programmingLanguage.isUTGSupported()) {
-            extractSupplementalFileContextForTst(psiFile, targetContext)
+            if (CodeWhispererUserGroupSettings.getInstance().getUserGroup() == CodeWhispererUserGroup.Utg) {
+                extractSupplementalFileContextForTst(psiFile, targetContext)
+            } else {
+                emptyList()
```

Comment:
not returning null for purpose, null is returned only for non-supported language. We will send supplemental context telemetry no matter which user group the user belongs